htdocs/core/triggers/interface_50_modAgenda_ActionsAuto.class.php

```diff
@@ -475,6 +475,17 @@ public function runTrigger($action, $object, User $user, Translate $langs, Conf
 				$object->actionmsg = $langs->transnoentities("SupplierOrderSentByEMail", $object->ref);
 			}
 
+			// Parameters $object->sendtoid defined by caller
+			//$object->sendtoid=0;
+		} elseif ($action == 'ORDER_SENTBYMAIL') {
```

Comment:
This case seems already present at line 218

Reply:
My mistake! First it didn't work in my version. Fixed it with this code and setting the parameter, but it was already there!
Case closed!